pkg/api/compatibility/compatibility_tester.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package compatibility
```

Comment:
note specifically that I nested it under testing :)

On Tue, Sep 15, 2015 at 7:19 PM, Paul Morie <notifications@github.com>
wrote:

> In pkg/api/compatibility/compatibility_tester.go
> <https://github.com/kubernetes/kubernetes/pull/13939#discussion_r39588008>
> :
>
> > +Copyright 2015 The Kubernetes Authors All rights reserved.
> > +
> > +Licensed under the Apache License, Version 2.0 (the "License");
> > +you may not use this file except in compliance with the License.
> > +You may obtain a copy of the License at
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package compatibility
>
> Compat works for me. I don't remember the exact cycle but I can dig it up
> if you need proof.
> … <#14fd3f317c53e4f2_>
> On Tuesday, September 15, 2015, Tim Hockin <notifications@github.com>
> wrote: In pkg/api/compatibility/compatibility_tester.go <
> https://github.com/kubernetes/kubernetes/pull/13939#discussion_r39585857>
> : > +Copyright 2015 The Kubernetes Authors All rights reserved. > + >
> +Licensed under the Apache License, Version 2.0 (the "License"); > +you may
> not use this file except in compliance with the License. > +You may obtain
> a copy of the License at > + > +
> http://www.apache.org/licenses/LICENSE-2.0 > + > +Unless required by
> applicable law or agreed to in writing, software > +distributed under the
> License is distributed on an "AS IS" BASIS, > +WITHOUT WARRANTIES OR
> CONDITIONS OF ANY KIND, either express or implied. > +See the License for
> the specific language governing permissions and > +limitations under the
> License. > +*/ > + > +package compatibility What cycle? How about
> testing/compat ? … <#14fd3c42e865fe8c_> On Tue, Sep 15, 2015 at 6:19 PM,
> Paul Morie <notifications@github.com <javascript:_e(%7B%7D,'cvml','
> notifications@github.com');>> wrote: In
> pkg/api/compatibility/compatibility_tester.go <
> https://github.com/kubernetes/kubernetes/pull/13939#discussion_r39585472>
> : > +Copyright 2015 The Kubernetes Authors All rights reserved. > + >
> +Licensed under the Apache License, Version 2.0 (the "License"); > +you may
> not use this file except in compliance with the License. > +You may obtain
> a copy of the License at > + > +
> http://www.apache.org/licenses/LICENSE-2.0 > + > +Unless required by
> applicable law or agreed to in writing, software > +distributed under the
> License is distributed on an "AS IS" BASIS, > +WITHOUT WARRANTIES OR
> CONDITIONS OF ANY KIND, either express or implied. > +See the License for
> the specific language governing permissions and > +limitations under the
> License. > +*/ > + > +package compatibility @thockin
> <https://github.com/thockin> <https://github.com/thockin> <
> https://github.com/thockin> Import cycle :-/ — Reply to this email
> directly or view it on GitHub <
> https://github.com/kubernetes/kubernetes/pull/13939/files#r39585472>. —
> Reply to this email directly or view it on GitHub <
> https://github.com/kubernetes/kubernetes/pull/13939/files#r39585857>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13939/files#r39588008>.
>